customer-web-api/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    compile project(":common-swagger")
```

Comment:
Which of these dependencies are needed?
Please review other build.gradle files.